recipes/python-kaleido/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "python-kaleido" %}
+{% set version = "0.1.0a5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido-{{ version }}.tar.gz
+  sha256: 55ff116d8a05998fd289f9480e30f87a4a24c14bc9f530cb2ee6de0c65c4a4f4
+
+build:
+  number: 0
+  noarch: python
+  script: |
+    cp {{ environ["RECIPE_DIR"] }}/LICENSE.txt .
+    {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - kaleido-core ={{ version }}
+
+test:
+  requires:
+    - plotly >=4.10,<5.0
+  imports:
+    - kaleido
+
+about:
+  home: https://github.com/plotly/Kaleido
+  license: MIT
+  license_family: MIT
+  license_file: {{ environ["RECIPE_DIR"] }}/LICENSE.txt
```

Comment:
Should be able to just do `LICENSE.txt`.

Reply:
:+1: